### ISSUE TITLE
Add mining mastery power

### ISSUE THREAD

**Issue by lukeacraven** [RamanujansGhost/s87powers#13]
This power would mine out a 3x2x2 or 2x2x3 area of only stone depending on the player's orientation.  It would probably cost 1 lapis lazuli per area mined (12 blocks).  This power would need to appropriately wear the tool out, or perhaps even wear it out at 1.25x the speed (15 blocks of damage per area/12 blocks).  Additionally, this power would need to respect faction territory.  It could be toggled, or it could be performed when a player with the power right clicks stone while wielding a pickaxe.

**Comment by BlipRanger** [comment]
If this is a "human" power, we should consider making it slightly weaker/slower, perhaps a lower area.

**Comment by lukeacraven** [comment]
Yes, in fact, I am fine with a mining mastery power, but there is likely an easier/more appropriate way to do it, such as a small chance of a fortune effect triggering when ore is mined.
